src/config/check_element.c

```diff
@@ -16,12 +16,87 @@
 ** [Note] On success or on failure, close the file descriptors
 */
 
-void	check_texture()
+void	check_texture(char **element)
 {
+	struct stat	path_stat;
 
+	if (stat(element[1], &path_stat) != 0 || !S_ISREG(path_stat.st_mode))
+	{
+		ft_free_split(&element);
+		exit_error(EPATH);
+	}
```

Comment:
This functions (`stat`, `S_ISREG`) are not allowed by the subject of the project.

The function `mlx_xpm_file_to_image` already verifies if the xpm is valid, so I propose to delete this section.